composer.json

```diff
@@ -11,7 +11,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.4",
-        "pestphp/pest": "^0.2",
+        "pestphp/pest": "^0.3",
```

Comment:
Can you update the master alias? So this project can be included?

Reply:
Sorry, I'm not sure what you mean. 🤔

Reply:
https://github.com/pestphp/pest/blob/439ebcdcaf6607ab875da8c0ce453e05b5b2b096/composer.json#L75

Reply:
Oh right, so just add a branch alias to this? Cool. 👍🏻